internal/internal_task_handlers.go

```diff
@@ -1240,7 +1240,8 @@ func (ath *activityTaskHandlerImpl) Execute(taskList string, t *s.PollForActivit
 	activityImplementation := ath.getActivity(activityType.GetName())
 	if activityImplementation == nil {
 		// Couldn't find the activity implementation.
-		return nil, fmt.Errorf("unable to find activityType=%v", activityType.GetName())
+		supported := strings.Join(ath.hostEnv.getRegisteredAcvitityNames(), ", ")
```

Comment:
It is kind of weird that above ath.getActivity is called and getRegisteredActivityNames added to hostEnv instead of activityTaskHandlerImpl.

Reply:
fixed.